src/code/qualification/plotQualificationDDIRatio.m

```diff
@@ -91,6 +97,11 @@
                 break
             end
         end
+        if isempty(findPathOutput)

```

Comment:
@pchelle do you also check somewhere if `SimResultControl.outputPathList` and `SimResultDDI.outputPathList` are not empty? Otherwise line 100 and/or line 123 will crash.